scripts/create-component/plop-templates-component/src/components/{{componentName}}/{{componentName}}.tsx.hbs

```diff
@@ -6,7 +6,7 @@ import { use{{componentName}}Styles } from './use{{componentName}}Styles';
 
 /**
  * Define a styled {{componentName}}, using the `use{{componentName}}` and `use{{componentName}}Styles` hooks.
- * {@docCategory {{componentName}}\}
+ * {@docCategory {{componentName}} }
```

Comment:
Did this come through literally as `\}` before?

Reply:
yes, I always had no idea why it was there and it actually braeks docgen

Reply:
Yeah--it seems like a bug in plopfile because there doesn't seem to be a way to include a closing brace `}` without at least a space before it. But at least this should generate valid docgen format.